src/psyclone/transformations.py

```diff
@@ -1604,9 +1617,12 @@ class ACCParallelTrans(ParallelRegionTrans):
     >>> newschedule, _ = dtrans.apply(newschedule)
     >>> newschedule.view()
     '''
-    from psyclone import gocean1p0, psyGen
+    from psyclone import gocean1p0, nemo, psyGen
     valid_node_types = (gocean1p0.GOLoop, gocean1p0.GOKern,
-                        psyGen.ACCLoopDirective)
+                        nemo.NemoLoop, nemo.NemoKern, nemo.NemoIfBlock,
```

Comment:
I can't see how this approach can continue if we start adding in kernel code as the list will get massive. OK for now though.

Reply:
Ah, that's a good point that I hadn't thought of. We can always use base classes but we'll see...